### ISSUE TITLE
Demo for external providers in media embed does not show embeds

### ISSUE THREAD

**Issue by FilipTokarski** [ckeditor/ckeditor5#15539]
## 📝 Provide a description of requested docs changes

As in the title.
1. [Go to demo](https://ckeditor.com/docs/ckeditor5/latest/features/media-embed.html#using-external-services-for-preview)
2. Embeds are not displayed

![Screenshot 2023-12-15 at 12 53 27](https://github.com/ckeditor/ckeditor5/assets/34380544/b819b1bb-7abc-4b6f-981b-9837ea9ab07a)

**Comment by gorzelinski** [comment]
I checked the mentioned demo. It appears that the problem lies in this external script - iframely. I checked the demo in the latest and some previous versions of our docs - previews don't work. The problem is also browser-independent. On the same page, there is another demo without this script, and it works as expected.

I tried to remove troublesome attributes like "allow autoplay," but it didn't help. For now, the best solution I come up with is to add a custom parameter to the script.

```html
<script async charset="utf-8" src="//cdn.iframe.ly/embed.js?cancel=0"></script>
```

The parameter will recover URLs with no preview as regular links. It's not the best solution, but the media embed blocks don't disappear.
